web/src/lib/Movie/MovieResult.svelte

```diff
@@ -3,24 +3,24 @@
   import { invalidateAll } from '$app/navigation';
   import { FontAwesomeIcon } from '@fortawesome/svelte-fontawesome';
   import { faPlusCircle } from '@fortawesome/free-solid-svg-icons';
-  import type { TMDBMovieResult } from '../../ambient';
+  import type { MovieType } from '../../ambient';
   import Image from '$lib/Image.svelte';
   import Label from './Label.svelte';
   import { getPosterUrl } from '../../utils';
   import { addToast } from '../../store';
   import { addedError, addedSuccess } from './messages';
 
-  export let movie: TMDBMovieResult;
+  export let movie: MovieType;
   export let listId: string;
   export let showMovieModal: boolean;
 </script>
 
 <div class="w-96 h-80 flex rounded-xl bg-background p-3 flex-shrink-0 mb-5">
   <div class="relative w-2/5 mr-3 rounded-md">
-    <div><Image src="{getPosterUrl(movie.poster_path)}" watched="{true}" /></div>
-    <p class="pt-2"><Label>rating:</Label> {movie.vote_average.toFixed(1)}</p>
+    <div><Image src="{getPosterUrl(movie.posterUrl)}" watched="{true}" /></div>
```

Comment:
I'm getting nulls when trying to build the posterurl url: `https://image.tmdb.org/t/p/w500null` in console from the movie search results modal, but the posters are still appearing in the search results.